jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/excel/ExcelExporter.java

```diff
@@ -264,6 +265,19 @@ public void exportDataGrid(Downloader downloader, Grid<Object> dataGrid, ExportM
         }
     }
 
+    protected String getColumnHeaderCaption(DataGrid.Column<?> column) {
+        String caption = column.getHeaderText();
+        if (caption != null) {
```

Comment:
`!Strings.isNullOrEmpty(...)`

Reply:
I think in this case empty value is also valid value to return but null means that we should check header component to obtain some header text

Reply:
It's better to check empty value, because Vaadin in many cases doesn't support `null` and empty means `null`.